### DISCUSSION TITLE
frigate/<camera_name>/<object_name> should report 0

### DISCUSSION THREAD

**Discussion by tbclark3** [blakeblackshear/frigate#3979] [Ideas]
frigate/<camera_name>/<object_name> correctly reports when the number of objects exceeds 0, but doesn't report when the number changes back to 0, which is just as important.  There should be an mqtt update when the object count is 0.

**Comment by NickM-27** [comment]
There is an update to go back to 0.

<img width="535" alt="Screen Shot 2022-09-29 at 8 13 17 AM" src="https://user-images.githubusercontent.com/14866235/193055237-ab334703-cab7-4889-8bbf-e488a86e5e64.png">

If you give more info to describe what is actually happening we can hopefully understand what is going on in your case.
